dev/tools/make-changelog.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
-echo "PR number"
+printf "PR number? "
 read -r PR
 
-echo "Where? (type a prefix)"
+printf "Where? (type a prefix)\n"
```

Comment:
What's wrong with echo?

Reply:
I wanted to avoid going to the next line (in the first question), but the `echo -n` variant does not exist on MacOS X.

Reply:
Please don't use printf for the other prints then

Reply:
Somehow, `printf` is generally recommended over `echo` (see e.g. this [post](https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo)) and this fits my experience. So, I would lean to suggest that we keep `printf`.